### DISCUSSION TITLE
几点建议

### DISCUSSION THREAD

**Discussion by HRS2049** [tamlok/vnote#2006] [Ideas]
刚刚用了一天的VNote，爱上它了。以下提议，难免肤浅，如有不对，望海涵。我只是一个使用app的非斜杠中年。
1. VNote可以使用正则，一下子就把很多类似的app甩了几条街；可是目前它不具备存储正则的功能，每次都要重新输入，如果可以记住用户输入的正则就好了；同时，在用户需要清除的时候，可以一条，几条，或全部清理掉，就完美了。
2. 搜索功能，不管用正则还是一般查找/替换，目前看起来只能在一份笔记里面进行，强烈建议可以选择：在当前笔记，在当前笔记本，在指定（几个）笔记（本）之间选择，这样，可以充分发挥查找/替换的功能；
3. 导出：强烈建议增加EPUB导出功能。这样方便把自己的笔记放到手机或电纸书阅读器里看，这是做笔记的初心。毕竟，PDF等格式在手机等移动设备里的阅读体验实在不好；
4. 导出pdf，目前看起来是每一份笔记一个pdf文档，是否可以实现一次性导出一个笔记本内的所有笔记为一个完整的pdf档？

**Comment by tamlok** [comment]
左边导航栏那里多点一下，有全文搜索，导出可以选择多合一导出一个笔记本为一个文件，可以使用pandoc来导出任意格式，具体参考文档。

**Reply by HRS2049** [reply]
感谢大神。不好意思，我应该先看看那两个修养的视频，就不会这样问了。微信赞赏，必须的。
